cluster-api-gcp/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing Guidelines
+
+## Developing
+
+When making changes to the machine controller, it's generally a good idea to delete any existing cluster created with an older version of the cluster-api.
+
+```
+$ ./cluster-api-gcp delete
```

Comment:
Consider starting with "git clone", "cd <some-dir>" , and build steps for someone that might be totally new to the process.

Reply:
Good idea. Done.